### DISCUSSION TITLE
Azure Cosmos DB connection timeout

### DISCUSSION THREAD

**Discussion by stephenstroud** [strapi/strapi#6456] [Legacy Topics]
**Node.js version**:
v12.16.2
**npm version**:
6.13.6
**Strapi version**:
3.0.1
**Operating system**:
Windows 10

**Which example is causing problem?**

yarn create strapi-app my-project 
CustomDB option: Mongo

I'm trying to connect to Azure CosmosDB through the supported Mongodb connector, however, I'm getting a timeout error. The connection string details are correct.

Configuration:
![image](https://user-images.githubusercontent.com/7649287/83328648-7ae9ab80-a27c-11ea-838d-e5b8dc3568ca.png)

Azure CosmoDB configuration

![image](https://user-images.githubusercontent.com/7649287/83328671-b5ebdf00-a27c-11ea-8293-ed927ae66a20.png)

Error:
![image](https://user-images.githubusercontent.com/7649287/83328694-d87df800-a27c-11ea-8ed3-df6bb1a15eb6.png)

Have I missed something here?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
